generator/generator-opc-generator/generators/app/templates/src/index.js

```diff
@@ -0,0 +1,51 @@
+// @ts-nocheck
+import { LitElement, html } from "lit-element";
+import style  from './styles/style.scss';
+
+export class MyCounter extends LitElement {
```

Comment:
Why are we creating MyCounter element?

Reply:
The class name is now being created dynamically <%= componentClass %>